client/Components/GetInTouchForm.js

```diff
@@ -91,24 +113,20 @@ const GetInTouchForm = ( { handleFormChange }) => {
           <div className="form-item cell">
             <div className="cell-wrap">
               <label htmlFor="moveInDateBtn">Move-in date</label>
-              <button id="moveInDataBtn" type="button" className="move-in">Jan 21</button>
-              <input
-                id="moveInDate"
-                name="moveInDate"
-                hidden
-                type="text" />
+              <button id="moveInDataBtn" type="button" className="move-in" onClick={handleMoveInClick}>Jan 21</button>
             </div>
+            <input id="calendar" type="date" hidden/>
           </div>
           <div className="form-item cell">
             <div className="cell-wrap">
               <label htmlFor="desiredCapacity">Number of People</label>
               <input
                 id="desiredCapacity"
                 type="number"
-                defaultValue="1"
-                required></input>
-              <button>-</button>
-              <button>+</button>
+                defaultValue={numberOfPeople}
```

Comment:
you might want to have this input update the numberOfPeople when user types in number too